test/controllers/income/bbsi/BbsiControllerSpec.scala

```diff
@@ -122,18 +125,40 @@ class BbsiControllerSpec extends PlaySpec
       val sut = createSUT
       when(sut.bbsiService.bankAccount(any(), any())(any())).thenReturn(Future.successful(Some(bankAccount)))
 
+      when(sut.journeyCacheService.currentValue(any())(any())).thenReturn(Future.successful(None))
+
       val result = sut.decision(1)(RequestBuilder.buildFakeRequestWithAuth("GET"))
       status(result) mustBe OK
 
       val doc = Jsoup.parse(contentAsString(result))
       doc.title() must include(Messages("tai.bbsi.decision.title", bankAccount.bankName.getOrElse("")))
     }
 
+    "show decision page with some saved cache data" in {
```

Comment:
Could possibly clarify this a bit more ..
"show decision page with a cached user choice pre-selected"  - something along those lines?